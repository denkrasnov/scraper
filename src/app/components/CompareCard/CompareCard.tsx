import React, { FC } from "react";

import Image from "src/atoms/Image";
import Card from "src/atoms/Card";
import Box from "src/atoms/Box";
import { H3 } from "src/atoms/Headings";
import Text from "src/atoms/Text";
import Divider from "src/atoms/Divider";

const ProductCard: FC = () => {
  return (
    <Card>
      <Box>
        <Box width="216px">
          <Image
            alt="product photo"
            borderRadiusLeft
            objectFit="contain"
            src="https://via.placeholder.com/350x150"
          />
        </Box>
        <Box flexDirection="column" padding="0 10px 10px 10px" width="100%">
          <H3>Product name</H3>
          <Box margin="0 0 15px">
            <Text>Description here</Text>
          </Box>
          <Divider />
          <Box justifyContent="flex-end" margin="15px 0px 5px 0">
            <Text color="MAIN_RED" fontSize="fs400">
              Price here
            </Text>
          </Box>
        </Box>
      </Box>
    </Card>
  );
};

export default ProductCard;
