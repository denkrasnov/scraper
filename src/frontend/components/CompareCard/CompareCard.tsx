import React, { FC } from "react";

import Image from "~app/atoms/Image";
import Card from "~app/atoms/Card";
import Box from "~app/atoms/Box";
import { H3 } from "~app/atoms/Headings";
import Text from "~app/atoms/Text";
import Divider from "~app/atoms/Divider";

const ProductCard: FC = () => {
  return (
    <Box backgroundColor="WHITE" display="block" maxWidth="738px">
      <Card>
        <Box>
          <Box width="216px">
            <Image
              alt="product photo"
              borderRadiusLeft
              objectFit="contain"
              src="https://via.placeholder.com/350x150"
            />
          </Box>
          <Box flexDirection="column" padding="0 10px 10px 10px" width="100%">
            <H3>Product name</H3>
            <Box margin="0 0 15px">
              <Text>Description here</Text>
            </Box>
            <Divider />
            <Box justifyContent="flex-end" margin="15px 0px 5px 0">
              <Text color="MAIN_RED" fontSize="fs400">
                Price here
              </Text>
            </Box>
          </Box>
        </Box>
      </Card>
    </Box>
  );
};

export default ProductCard;
