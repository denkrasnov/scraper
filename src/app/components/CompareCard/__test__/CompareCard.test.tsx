import React from "react";
import { shallow } from "enzyme";

import CompareCard from "..";

describe("CompareCard", () => {
  it("renders", () => {
    const component = shallow(<CompareCard />);
    expect(component).toMatchSnapshot();
  });
});
