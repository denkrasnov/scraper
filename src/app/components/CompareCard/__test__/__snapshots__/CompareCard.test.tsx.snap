// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CompareCard renders 1`] = `
<Card>
  <Box>
    <Box
      width="216px"
    >
      <Image
        alt="product photo"
        borderRadiusLeft={true}
        objectFit="contain"
        src="https://via.placeholder.com/350x150"
      />
    </Box>
    <Box
      flexDirection="column"
      padding="0 10px 10px 10px"
      width="100%"
    >
      <Component>
        Product name
      </Component>
      <Box
        margin="0 0 15px"
      >
        <Text>
          Description here
        </Text>
      </Box>
      <Divider />
      <Box
        justifyContent="flex-end"
        margin="15px 0px 5px 0"
      >
        <Text
          color="MAIN_RED"
          fontSize="fs400"
        >
          Price here
        </Text>
      </Box>
    </Box>
  </Box>
</Card>
`;
